python/kfserving/setup.py

```diff
@@ -12,32 +12,58 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from setuptools import setup, find_packages
+import setuptools
 
-tests_require = [
-    'pytest',
-    'pytest-tornasync',
-    'mypy'
+REQUIRES = [
+    'certifi >= 14.05.14',
+    'six >= 1.10',
+    'python_dateutil >= 2.5.3',
+    'setuptools >= 21.0.0',
+    'urllib3 >= 1.15.1',
+    'kubernetes >= 9.0.0',
+    'tornado >= 1.4.1',
+    'argparse >= 1.4.0',
+    'minio >= 4.0.9',
+    'google-cloud-storage >= 1.16.0',
+    'azure-storage-blob >= 2.0.1',
+    'numpy',
 ]
 
-setup(
-    name='kfserver',
-    version='0.1.0',
+setuptools.setup(
+    name='kfserving',
+    version='0.0.2',
+    author="Kubeflow Authors",
     author_email='ellisbigelow@google.com',
     license='../../LICENSE.txt',
```

Comment:
You may want to add your own email as author/maintainer for this - so that we know who to reach out incase things don't work :-)

Reply:
Added, thanks!